saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -198,8 +206,12 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
         channel = cls.clean_channel_id(info, instance, cleaned_input, channel_id)
 
         voucher = cleaned_input.get("voucher", None)
+        voucher_code = cleaned_input.get("voucher_code", None)
+        cls.clean_voucher_and_voucher_code(voucher, voucher_code)
         if voucher:
             cls.clean_voucher(voucher, channel)
```

Comment:
We should also assign `voucher_code` when the `voucher` is provided. We should take the first `voucher_code` and assign

Reply:
done